core/libp2p/multi/multistream.hpp

```diff
@@ -6,11 +6,37 @@
 #ifndef KAGOME_MULTISTREAM_HPP
 #define KAGOME_MULTISTREAM_HPP
 
+#include <filesystem>
+
+#include "common/buffer.hpp"
+#include "common/result.hpp"
+
 namespace libp2p::multi {
   /**
    * Format of stream identifier used in Libp2p
    */
-  class Multistream {};
+  class Multistream {
+
+  public:
+
+    static auto create(std::filesystem::path codecPath,
```

Comment:
this std::filesystem::path confuses a bit...
maybe it is good to hide under alias (`using`)

also, add docs for public API

Reply:
The same here, multistream doesn't belong here, I just got carried away by one problem here and forget to remove it.

Reply:
okay